content/updates/2020-07/1-associations-with-quantities.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_img: ./img/ryland-extending-dining-table-and-4-chairs-set-walnut-and-black_made.com.png
+pim_announcement_alt_img: "New association type with quantities"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New: association type with quantities
+::: meta-data type="New" features="Product Associations" available="early July" link-to-doc="../articles/manage-your-association-types.html#create-an-association-type-with-quantities"
+:::
+
+With the new association type with quantities, you are now able to create a product set composed of other products that are associated with different quantities for each one.
+
+For example, it is now possible to create a dining table set composed of 1 table and 4 chairs, just like this one (picture courtesy of Made.com):
+
+![This is a product set composed of different associated products with quantities](../img/ryland-extending-dining-table-and-4-chairs-set-walnut-and-black_made.com.png)
```

Comment:
Hey Felix, could you also add the render of this pack in the PIM to make it more visual and impactful to users? 
Ie how this great Made.com product set is configured in Akeneo, so a screenshot with the table, the chairs and the quantities.
It's important to show how it's done in Akeneo. 
Thank you!

Reply:
Hello Laure, I didn't have the possibility to display the image from Made.com dure to IP issue. So to meet your request I had to take pictures of my own furniture (less glamour) to create a real dining set with a table and 4 chairs. I've created them in the PIM also.
What do you think?